zip.sh

```diff
@@ -18,5 +18,6 @@ else
 	exit 1
 fi
 
-zip -r "$zip_file_name" "$plugin_slug" -x@"$ignore_file"
-mv "$zip_file_name" "$plugindir"
\ No newline at end of file
+# Ensure every line in the ignore file begins and ends with *
+sed "s/^[^*]/*&/g" "$ignore_file" | sed "s/[^*]$/&*/g" | xargs zip -r "$zip_file_name" "$plugin_slug" -x
```

Comment:
Instead of passing the ignore file with `-x@"$ignore_file"`, this pipes the contents of the file with `xargs`